react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+
+export default function AppDrawer() {
+  const [isHidden, setIsHidden] = useState(true);
+
+  const games = {
+    hidden: {
+      popUpBackground: '',
+      left: 'left'
+    },
+    shown: {
+      popUpBackground: 'pop-up-background',
+      left: ''
+    }
+  };
+
+  function popUpMenu() {
+    setIsHidden(!isHidden);
+  }
+
+  function selectGame(event) {
+    const closestGame = event.target.closest('a');
+    if (closestGame) {
+      setIsHidden(!isHidden);
+    }
+  }
+
+  function backgroundClose() {
+    setIsHidden(!isHidden);
+  }
+
+  const classes = games[!isHidden ? 'shown' : 'hidden'];
+  return (
+    <div className='container'>
+      <div className={classes.popUpBackground} onClick={backgroundClose}></div>
+      <div className='relative' onClick={selectGame}>
+        <div className='absolute'>
+          <i className='fa-solid fa-bars' onClick={popUpMenu}></i>
+        </div>
+        <div className={`absolute menu row ${classes.left}`}>
+          <h1>Choose a Game</h1>
```

Comment:
Instead of capturing the click up in the `div` on line 36, capture the click in each of these `a` tags and pass in the game. Something like:
`<a onClick={() => selectGame('Escape from Tarkov')}>Escape from Tarkov</a>`

You can see that if you do this you'll want the game in a variable. Which will probably lead you to putting all the games into an array.